### PR TITLE
:memo: Rework CDN docs

### LINKED ISSUE #2670: 📝 Rework CDN docs
### Where on docs.platform.sh should be changed?

https://docs.platform.sh/domains/cdn.html
https://docs.platform.sh/domains/cdn/fastly.html
https://docs.platform.sh/domains/cdn/cloudflare.html

### What exactly should be updated?

Update for:

- Content accuracy
- Includes what it should
- Doesn't include what doesn't fit
- Fits template
- Fits style guide

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by AnouckColson**
## Why

Closes #2670 

## What's changed

Rework CDN pages

### CODE REVIEW

File: docs/src/domains/cdn/_index.md [hunk 1]
```diff
@@ -1,183 +1,171 @@
...
+However, depending on your needs, you can [install your own WAF](../../security/waf.md) 
+and add security measures not included in your plan.  
+
+## Create a CNAME record for your domain name
+
+To start routing client traffic through your CDN,
+create `CNAME` records for your domain names through your DNS provider.
+For more information, see you DNS provider's official documentation.
+
+Note that `CNAME` records can't point to apex domains.
```
Comment (AnouckColson) [inline docs/src/domains/cdn/_index.md:41]: When digging on `CNAME` records, all the resources I came across stated that `CNAME` records can't point to apex domains, but that there are workarounds provided by DNS providers. So the assumption in our docs that a registrar might actually support 'CNAME' records for apex domains by default seems to be wrong. It seems it will always be through a workaround.

File: docs/src/domains/cdn/cloudflare.md [hunk 2]
```diff
@@ -1,69 +1,52 @@
...
-mostly like using your project's Let's Encrypt certificate.
+To do so, from your Cloudflare account, [enable Full (strict) SSL]https://developers.cloudflare.com/ssl/origin-configuration/ssl-modes/full/.
+All HTTP requests are then automatically redirected to HTTPS.
 
-```text
-# Cloudflare's Full SSL option
-          https                       https
-User <---------------> Cloudflare <-------------> Platform.sh
-```
+## Renew your TLS certificates
```
Comment (AnouckColson) [inline docs/src/domains/cdn/cloudflare.md:27]: I couldn't verify the information in this section. I updated it by checking if the settings mentioned were still available in the Cloudflare interface, but I honestly have no idea if this is up-to-date or actually needed 🤷‍♀️ . Does it ring a bell?

File: docs/src/domains/cdn/cloudflare.md [hunk 3]
```diff
@@ -1,69 +1,52 @@
...
+- Remove all domains pointing to Cloudflare from your Platform.sh project before the renewal process.
+- Go to your Cloudflare Console and follow these steps:
 
-In short: *Always use **Full SSL** unless you have a very clear reason to do otherwise*
+  1. Go to **SSL/TLS**>**Edge Certificates** and turn off **Always Use HTTPS**.
+  2. [Create a page rule](https://support.cloudflare.com/hc/en-us/articles/218411427-Understanding-and-configuring-Cloudflare-Page-Rules-Page-Rules-Tutorial-#h_38Gq7mduJiXIjpVLxp3q19) for `/.well-known/acme-challenge/` with **SSL** set to **off**.
+  3. Create a second page rule for `*` that turns **Always Use HTTPS** back on.
 
-## Let's Encrypt certificate renewal
+## Use Fastly CNAME flattening
```
Comment (AnouckColson) [inline docs/src/domains/cdn/cloudflare.md:44]: I moved this section to mirror the page dedicated to Fastly.

File: docs/src/domains/cdn/fastly.md [hunk 4]
```diff
@@ -1,58 +1,55 @@
...
-but it is a common choice for customers to self-configure.
+By default, a Fastly CDN is included in Dedicated projects.
+Self-Service Grid plans don't automatically include a Fastly CDN, 
+but you can set up one at any time.
 
-Launching a Platform.sh site with Fastly in front of it is nearly the same as launching normally.
-There are only a few notable differences.
-Individual apps may have their own Fastly setup instructions or additional modules.
-Consult the documentation for your app for specific details.
+## Get a DNS TXT record
```
Comment (AnouckColson) [inline docs/src/domains/cdn/fastly.md:16]: This is for Dedicated plans only, but it's not a feature so the tier availability header wouldn't work here. Is there specific formatting you would use in this case to signal that not all clients are concerned?

File: docs/src/domains/cdn/fastly.md [hunk 5]
```diff
@@ -1,58 +1,55 @@
...
 
-Generally, Platform.sh recommends specifying only HTTPS routes in your `routes.yaml` file.
-This results in all pages being served over SSL and any requests for an HTTP URL automatically redirected to HTTPS.
-When Fastly has been added as an intermediary to your project, however, this automatic redirect fails.
-Once a user visits your site with an HTTP request and is directed to Fastly,
-Fastly continues to query Platform.sh using HTTPS (since that is how origin has been configured),
-resulting in Platform.sh never detecting the HTTP request in the first place.
-Enabling the "Force TLS and enable HSTS" option causes Fastly to do the same HTTP to HTTPS redirect as Platform.sh,
-ensuring that all requests are over HTTPS end-to-end.
+To do so, from your Fastly account, [force TLS and enable HSTS](https://docs.fastly.com/en/guides/enabling-hsts-through-fastly#forcing-tls-and-enabling-hsts).
```
Comment (CollierCZ) [inline docs/src/domains/cdn/fastly.md:42]: :clipboard: 
```suggestion
To do so, in your Fastly account [force TLS and enable HSTS](https://docs.fastly.com/en/guides/enabling-hsts-through-fastly#forcing-tls-and-enabling-hsts).
```

Comment (CollierCZ) [reply docs/src/domains/cdn/fastly.md:42]: Might also be worth explaining what HSTS is? Not sure, if it's explained enough at the link.

File: docs/src/domains/cdn/fastly.md [hunk 6]
```diff
@@ -1,58 +1,55 @@
...
+All HTTP requests are then automatically redirected to HTTPS.
 
-## DNS TXT records
+## Use Cloudflare Anycast options for your apex domain
```
Comment (CollierCZ) [inline docs/src/domains/cdn/fastly.md:45]: :clipboard: 
```suggestion
## Use Fastly anycast options for your apex domain
```

Comment (CollierCZ) [reply docs/src/domains/cdn/fastly.md:45]: :hammer_and_wrench: As with Cloudflare
```suggestion
## Handle apex domains
```

File: docs/src/domains/cdn/fastly.md [hunk 7]
```diff
@@ -1,58 +1,55 @@
...
-If using the Fastly CDN that is included with a Platform.sh Enterprise subscription,
-you need to obtain a DNS TXT record from your Customer Support Engineer prior to going live.
-You need to enter that as a DNS TXT record with your domain registrar.
-This step should be done well in advance of the actual go-live.
+To start routing client traffic through Fastly,
+you need to [create `CNAME` records for your domain names](../../domains/steps/dns.md#why-cname-records) 
+through your DNS provider.
 
-## Anycast
+However, `CNAME` records can't point to apex domains.
```
Comment (CollierCZ) [inline docs/src/domains/cdn/fastly.md:51]: :clipboard: 
```suggestion
But `CNAME` records can't point to apex domains.
```

File: docs/src/domains/cdn/fastly.md [hunk 8]
```diff
@@ -1,58 +1,55 @@
...
+As a workaround, Fastly offers [Anycast options](https://docs.fastly.com/en/guides/using-fastly-with-apex-domains).
```
Comment (CollierCZ) [inline docs/src/domains/cdn/fastly.md:52]: :clipboard: 
```suggestion
As a workaround, Fastly offers [anycast options](https://docs.fastly.com/en/guides/using-fastly-with-apex-domains).
```

**Review by CollierCZ — CHANGES_REQUESTED** [review]
I think the explanations for things are much clearer and I'm glad you were able to get rid of unnecessary things and verify others. That's most of the work done.

I do think the structure of it could be clearer. The goals of each page aren't completely clear to me and so I don't know how each section fits there. Does that make sense? I think maybe the two child pages could be straight-forward how-to guides and the parent more conceptual, though I'm not sure if that's best. Just trying to brainstorm possibilities, since it came from something even more confusing. But a clearer structure would make the clear information even easier to digest.

File: docs/src/domains/cdn/fastly.md [hunk 9]
```diff
@@ -1,58 +1,55 @@
...
+
+To do so, follow these steps:
 
-Rather than create a DNS `CNAME` for your default Platform.sh branch (for instance `main-abcd123.abcdefgh1234567.eu.platformsh.site`),
-[configure Fastly](https://docs.fastly.com/guides/basic-configuration/working-with-domains)
-to respond to requests for your domain name and to treat the default Platform.sh branch as its backend server.
-Be sure to enable TLS for the backend connection to Platform.sh.
-Then configure your DNS to point your domain at Fastly instead.
-See the [Fastly documentation](https://docs.fastly.com/guides/basic-configuration/connecting-to-origins) for further details.
+1. [Enable TLS](./_index.md#set-up-tls-certificates) for the backend connection to Platform.sh.
```
[the remark this answers is not in the document]
Comment (AnouckColson) [reply docs/src/domains/cdn/fastly.md]: Looking closer at the Fastly and Cloudflare documentation, this looks like regular CDN setup and is not specific to Platform.sh, so I don't think it's worth having such information in our docs. Instead, I added a section with a link to Fastly and Cloudflare's respective Getting started docs that go through the whole setup.

File: docs/src/domains/cdn/cloudflare.md [hunk 10]
```diff
@@ -1,69 +1,71 @@
...
+S) certificate](../../other/glossary.md#transport-layer-security-tls).
 
-## Let's Encrypt certificate renewal
+TLS certificates are provided by trusted certificate authorities (CA) such as Let's Encrypt.
+If you use TLS certificates provided by Let's Encrypt,
+you need to renew them [every 90 days](https://letsencrypt.org/docs/faq/#what-is-the-lifetime-for-let-s-encrypt-certificates-for-how-long-are-they-valid). 
 
-Let's Encrypt expects the `.well-known` endpoint on all domains added.
-You have 2 options:
+To ensure such renewals succeed, remove all domains pointing to Cloudflare from your Platform.sh project
```
Comment (CollierCZ) [inline docs/src/domains/cdn/cloudflare.md:63]: :question: The `domains pointing to Cloudflare` part confused me a little because I started thinking that there was something in the Platform.sh project that was pointing to Cloudflare. That isn't true, though, right? I guess I just don't understand why this part is recommended, especially when the renewal is automated.

File: docs/src/domains/cdn/fastly.md [hunk 11]
```diff
@@ -1,58 +1,62 @@
...
+
+To use a Fastly CDN included in a Dedicated plan, 
+before going live you need to [request a DNS TXT record from support](../../overview/get-support.md#create-a-support-ticket).
+To add this DNS TXT record to your domain settings,
+contact your domain registrar.
+
+## 3. Set up your Fastly CDN
+
+To properly configure your Fastly CDN, 
+see the Fastly official documentation on [how to get started](https://docs.fastly.com/en/guides/getting-started#_basics).
```
Comment (CollierCZ) [inline docs/src/domains/cdn/fastly.md:35]: :question: Also a question here about whether to include the part about getting the target.

File: docs/src/domains/cdn/fastly.md [hunk 12]
```diff
@@ -1,58 +1,62 @@
...
+
+You need:
+
+- An up-and-running Platform.sh project
+- Self-service Grid plans: a [Fastly](https://www.fastly.com/) CDN subscription
 
 {{% disable-cache CDN="Fastly" %}}
 
-## Set the Platform.sh domain on Fastly
+## 2. Dedicated: Get a DNS TXT record
```
[the remark this answers is not in the document]
Comment (CollierCZ) [reply docs/src/domains/cdn/fastly.md:25]: :question: Is it only for Dedicated or also for Enterprise who aren't Dedicated?

**Review by CollierCZ — CHANGES_REQUESTED** [review]
I think the structure is much clearer now. So now I just have some questions about the technical details, now that I see how it fits together.

File: docs/src/domains/cdn/cloudflare.md [hunk 13]
```diff
@@ -1,69 +1,71 @@
...
+[enable HTTP strict transport security (HSTS)](https://developers.cloudflare.com/ssl/edge-certificates/additional-options/http-strict-transport-security/).
 
-This causes all traffic from Cloudflare to your project to be redirected to HTTPS,
-which sets off an endless loop as HTTPS traffic is presented as HTTP to your project no matter what.
+## 5. Let's Encrypt users: Renew your TLS certificates
 
-In short: *Always use **Full SSL** unless you have a very clear reason to do otherwise*
+To secure connections and prevent data leaks,
+authenticate your Platform.sh server with a [transport layer security (TL
+S) certificate](../../other/glossary.md#transport-layer-security-tls).
```
[the remark this answers is not in the document]
Comment (AnouckColson) [reply docs/src/domains/cdn/cloudflare.md:57]: Definitely not intentional

File: docs/src/domains/cdn/_index.md [hunk 14]
```diff
@@ -1,183 +1,163 @@
...
+
+## Host header forwarding
+
+When a HTTP request is made to a website, the client adds a `Host` header to the request. 
+The value of this header is the domain name the request is made to. 
+When a server hosts multiple websites, like what a CDN does,
+it can use the `Host` header to identify which domain to access to handle the request.
+
+When you use a CDN, the value of the `Host` header points to the CDN itself. 
+To allow the CDN edge server to direct the request to the appropriate Platform.sh server,
```
[the remark this answers is not in the document]
Comment (AnouckColson) [reply docs/src/domains/cdn/_index.md:66]: My understanding is that it's a bit of both, see updated explanation

File: docs/src/domains/cdn/cloudflare.md (hunk 10 above)
Comment (AnouckColson) [reply docs/src/domains/cdn/cloudflare.md:63]: Yes, I was [suspicious about this section](https://github.com/platformsh/platformsh-docs/pull/2746#discussion_r1047386499) from the get go, and your comments only add fuel to the idea that it should be removed.

File: docs/src/domains/cdn/_index.md [hunk 15]
```diff
@@ -5,179 +5,168 @@ weight: 3
...
+To add it to your CDN configuration, [create a support ticket](../../overview/get-support.md#create-a-support-ticket).
+
+## Host header forwarding
+
+When a HTTP request is made to a website, the client adds a `Host` header to the request. 
+The value of this header is the domain name the request is made to. 
+When a server hosts multiple websites, like what a CDN does,
+it can use the `Host` header to identify which domain to access to handle the request.
+
+When a request is made from a client for an object on a CDN edge server,
```
Comment (CollierCZ) [inline docs/src/domains/cdn/_index.md:70]: :pick: I feel like the `object` here isn't completely clear. At least, it made be confused at first. I wasn't sure what an `object` was in this context since it's not used elsewhere here. Not sure if there's something better though. :shrug:

File: docs/src/domains/cdn/_index.md [hunk 16]
```diff
@@ -5,179 +5,168 @@ weight: 3
...
-2. Get the IP range for your CDN provider:
-
+ 
+## Prevent direct access to your Platform.sh server
+ 
+When you use a CDN, there are three ways to prevent direct access to your Platform.sh server:
+ 
+- HTTP basic authentication
+- Allowing and denying IP addresses
+- Client-authenticated TLS
```
Comment (CollierCZ) [inline docs/src/domains/cdn/_index.md:105]: :question: With the table of contents on the right, is the list necessary? I don't think it hurts to have it, just wondering the benefit.

File: docs/src/domains/cdn/fastly.md [hunk 17]
```diff
@@ -1,55 +1,62 @@
...
+  - "/golive/steps/fastly.html"
 ---
 
-{{% description %}}
+You can [use a CDN](./_index.md) to deliver your site's content to users more quickly.
 
-A Fastly CDN is included for Dedicated projects.
-Platform.sh doesn't offer an integrated CDN on self-service Grid projects at this time,
-but it is a common choice for customers to self-configure.
+By default, a Fastly CDN is included in Dedicated projects.
```
Comment (CollierCZ) [inline docs/src/domains/cdn/fastly.md:12]: :lady_beetle: If the assumptions I made in the open question from the last review are true, then this entire page doesn't apply to Dedicated projects. But I don't think that is clear here. I think there should be something clearer about what Dedicated users need to do, if anything. I think it's only the Anycast support ticket, if applicable.

**Review by CollierCZ — CHANGES_REQUESTED** [review]
A few questions and issues left, including coordination with #2762

File: docs/src/domains/cdn/_index.md [hunk 18]
```diff
@@ -5,179 +5,168 @@ weight: 3
...
+an apex and a wildcard one.
+This allows for encryption of all traffic between your users and your app. 
+
+If you want to, you can also provide your own third-party TLS certificate.
+[Transfer your certificate](../../development/file-transfer.md), 
+its unencrypted private key and the intermediate certificate to a non-web accessible [mount](../../create-apps/app-reference.md#mounts), 
+on an environment that only Platform.sh support and trusted users can access.
+In this way, your private key can't be compromised.
+To add the TLS certificate to your CDN configuration, 
+[create a support ticket](../../overview/get-support.md#create-a-support-ticket).
```
[the remark this answers is not in the document]
Comment (AnouckColson) [reply docs/src/domains/cdn/_index.md]: My understanding was that it was available to anyone provided they paid an additional fee for it. But it also seems that we don't want to encourage users to use their own TLS certificates as most of them then forget to renew them (see context). Might be a bold choice, but if the certificates we provide are identical and don't present any risk of botched renewal, maybe it's best to remove the section entirely to avoid giving any ideas to clients? What do you think?

File: docs/src/domains/cdn/fastly.md (hunk 17 above)
Comment (AnouckColson) [reply docs/src/domains/cdn/fastly.md]: You are right in that most of the page isn't for Dedicated projects. However, there's information about CNAME records being provided by PSH that seems important to share with Dedicated users as it's specific to them. I moved it to the beginning of the page to clearly differentiate Dedicated from Grid (=> the rest of the page) but I'm not entirely sure it's the best place to put it.

File: docs/src/domains/cdn/_index.md (hunk 18 above)
[the remark this answers is not in the document]
Comment (CollierCZ) [reply docs/src/domains/cdn/_index.md]: The additional context seems to be only about certificates for Fastly when it's managed by us, and not a generic CDN. Maybe there should be a note on the Fastly part, or within a Dedicated section. I wouldn't remove just it because people forget to renew. But not sure it's relevant here, sure.

File: docs/src/domains/cdn/_index.md (hunk 15 above)
Comment (CollierCZ) [reply docs/src/domains/cdn/_index.md]: Yes, `resource` was clearer for me.

File: docs/src/domains/cdn/fastly.md [hunk 19]
```diff
@@ -1,55 +1,57 @@
...
 
-{{% description %}}
+You can [use a CDN](./_index.md) to deliver your site's content to users more quickly.
+To set up a Fastly CDN on a Grid project,
+follow the instructions on this page.
 
-A Fastly CDN is included for Dedicated projects.
-Platform.sh doesn't offer an integrated CDN on self-service Grid projects at this time,
-but it is a common choice for customers to self-configure.
+**Dedicated users**: Fastly CDNs included in Dedicated projects are set up by Platform.sh. If you need help with your Fastly CDN settings,
```
Comment (CollierCZ) [inline docs/src/domains/cdn/fastly.md:14]: :lady_beetle: This is the only instance of referring to `Dedicated users`. I think it'd be best to talk about Dedicated plans or projects or the like.

:hammer_and_wrench: This might be the first time someone sees that a Fastly CDN specifically is included in a Dedicated plan, so I might rephrase that part. Especially to emphasize that they're fully managed by Platform.sh (not just set up, but afterwards too).
```suggestion
**Dedicated plans** come with a Fastly CDN fully managed by Platform.sh.
To modify any settings for a managed Fastly CDN,
```

File: docs/src/domains/cdn/fastly.md [hunk 20]
```diff
@@ -1,55 +1,57 @@
...
+[open a support ticket](https://console.platform.sh/-/users/~/tickets/open).
+Note that when you request that a new domain is added to your Fastly service,
```
Comment (CollierCZ) [inline docs/src/domains/cdn/fastly.md:16]: :hammer_and_wrench: I feel like it might be clearer this way, but please let me know if you think I'm wrong. This might be less clear for non-native speakers. My problem is that `request that X is done` isn't really fully "correct", but the correct version involves subjunctives that most people will find difficult to parse.

```suggestion
When you request for a new domain to be added to your Fastly service,
```

File: docs/src/domains/cdn/fastly.md [hunk 21]
```diff
@@ -1,55 +1,57 @@
...
+Platform.sh support provides you with [a `CNAME` record](../../domains/steps/dns.md) for domain control validation.
```
Comment (CollierCZ) [inline docs/src/domains/cdn/fastly.md:17]: :pick: 
```suggestion
Platform.sh support provides you with a [`CNAME` record](../../domains/steps/dns.md) for domain control validation.
```

**Review by CollierCZ — CHANGES_REQUESTED** [review]
Just a few small things

**Review by AnouckColson — COMMENTED** [review]
@CollierCZ In my last draft, I included a section aimed at Dedicated users only at the top of the Fastly page. But with the latest review, I realized:

- There was too much content in the section, which imho messed up the introduction/didn't really fit there. As the existing Fastly page is a how-to one and is targeted at users who bring their own Fastly CDN, it didn't make much sense to me to move the **Dedicated users** section to another location on this page. This is why I created a separate page to tackle managed Fastly CDNs. Let me know if this split is OK with you.

- As I created this new page, I also realized that only talking about Dedicated users maybe wasn't the best/most accurate choice. Although managed Fastly CDNs are included by default in Dedicated projects, my understanding is (based on conversation linked in Context) that other types of users can opt for a managed Fastly CDN too. Which is why I steered away from the term Dedicated to include all potential users of managed Fastly CDNs. Again, let me know what you think.

File: docs/src/domains/cdn/managed-fastly.md [hunk 22]
```diff
@@ -0,0 +1,44 @@
...
+By default, Enterprise and Elite plans include two [TLS certificates](../../other/glossary.md#transport-layer-security-tls),
+an apex and a wildcard one.
+This allows for encryption of all traffic between your users and your app.
+ 
+If you use a Fastly CDN provided by Platform.sh, 
+you can provide your own third-party TLS certificates for an additional fee.
+[Transfer each certificate](../../development/file-transfer.md),
+its unencrypted private key, 
+and the intermediate certificate to a non-web accessible [mount](../../create-apps/app-reference.md#mounts),
+on an environment that only Platform.sh support and trusted users can access.
```
Comment (CollierCZ) [inline docs/src/domains/cdn/managed-fastly.md:35]: :pick: To help with the sentence length warning at the top. :shrug: 
```suggestion
If you don't have one, set up a [mount](../../create-apps/app-reference.md#mounts)
that isn't accessible to the web.
Use an environment with access limited to Platform.sh support and trusted users.
[Transfer](../../development/file-transfer.md) each certificate, its unencrypted private key, 
and the intermediate certificate to the mount.
```

File: docs/src/domains/cdn/fastly.md [hunk 23]
```diff
@@ -1,55 +1,55 @@
 ---
-title: "Fastly"
+title: "Set up your Fastly CDN"
+sidebarTitle: "Fastly setup"
 weight: 1
-description: In some cases you may want to opt to use a CDN such as Fastly rather than the Platform.sh router's cache. Using a CDN can offer a better time-to-first-byte for cached content across a wider geographic region at the cost of the CDN service.
+description: Learn how to configure your Fastly CDN.
+aliases:
+ - "/golive/steps/fastly.html"
```
Comment (CollierCZ) [inline docs/src/domains/cdn/fastly.md:7]: :clipboard: We shouldn't use aliases: https://github.com/platformsh/platformsh-docs/blob/main/CONTRIBUTING.md#redirect-pages

File: docs/src/domains/cdn/_index.md [hunk 24]
```diff
@@ -5,179 +5,131 @@ weight: 3
...
-
+ 
+Using a CDN speeds up the delivery of your site's content to its users.
+The CDN deploys edge servers at many locations around the world.
+These edge servers behave like local caches to nearby users.
+Bringing content closer to users helps enhance your site's perceived performance
+and so can improve user engagement and retention.
+
+Fastly is the recommended CDN for Platform.sh projects.
+By default, Dedicated projects include a [Fastly](./fastly.md) CDN managed by Platform.sh.
```
Comment (CollierCZ) [inline docs/src/domains/cdn/_index.md:16]: :lady_beetle: This refers to the managed version but links to the other version. Should be adjusted for the new page.

File: docs/src/domains/cdn/_index.md [hunk 25]
```diff
@@ -5,179 +5,131 @@ weight: 3
...
+Self-service Grid plans don't include a CDN by default, but you can set up one at any time.
```
Comment (CollierCZ) [inline docs/src/domains/cdn/_index.md:17]: :hammer_and_wrench: I might then add links here to the two set up pages 
```suggestion
Self-service Grid plans don't include a CDN by default, but you can set up one at any time,
such as for [Fastly](./fastly.md) or [Cloudflare](./cloudflare.md).
```

**Review by CollierCZ — CHANGES_REQUESTED** [review]
I think a separate page can certainly work. Just some thoughts related to that change left, then.

File: .platform/routes.yaml [hunk 26]
```diff
@@ -132,15 +132,17 @@ https://{default}/:
...
+
       # Deprecate Hugo aliases
       "/guides/general/mysql-replication.html":  { "to": "/add-services/mysql/mysql-replication.html", "code": 301 }
       "/dedicated.html":  { "to": "/dedicated-gen-2.html", "code": 301 }
       "/dedicated/(.*)":  { "to": "/dedicated-gen-2/$1", "code": 301, "regexp": true }
       "/guides/general/deploy-button.html":  { "to": "/development/deploy-button.html", "code": 301 }
       "/frameworks/deploy-button.html":  { "to": "/development/deploy-button.html", "code": 301 }
       "/development/public-ips.html":  { "to": "/development/regions.html", "code": 301 }
-      "/golive/steps/cloudflare.html":  { "to": "/domains/cdn/cloudflare.html", "code": 301 }
-      "/golive/steps/fastly.html":  { "to": "/domains/cdn/fastly.html", "code": 301 }
```
Comment (CollierCZ) [inline .platform/routes.yaml:143]: :question: What's the purpose of this change? The redirects already existed.

File: docs/src/domains/cdn/_index.md [hunk 27]
```diff
@@ -5,179 +5,132 @@ weight: 3
...
-
+ 
+Using a CDN speeds up the delivery of your site's content to its users.
+The CDN deploys edge servers at many locations around the world.
+These edge servers behave like local caches to nearby users.
+Bringing content closer to users helps enhance your site's perceived performance
+and so can improve user engagement and retention.
+
+Fastly is the recommended CDN for Platform.sh projects.
+By default, Dedicated projects include a [Fastly](./managed-fastly.md) CDN managed by Platform.sh.
```
Comment (CollierCZ) [inline docs/src/domains/cdn/_index.md:16]: :pick: To better fit the style guide and differentiate it from the following link.
```suggestion
By default, Dedicated projects include a [Fastly CDN managed by Platform.sh](./managed-fastly.md).
```

File: docs/src/domains/cdn/managed-fastly.md [hunk 28]
```diff
@@ -0,0 +1,48 @@
...
+---
+
+Instead of starting your own Fastly subscription and [managing your CDN yourself](./fastly.md),
+you can take advantage of a Fastly CDN provided by Platform.sh.
+For example, Dedicated projects include a managed Fastly CDN by default.
+These CDNs are exclusively set up and managed by Platform.sh.
+
+To modify any settings for a managed Fastly CDN,
+[open a support ticket](https://console.platform.sh/-/users/~/tickets/open).
+For information on how to subscribe to a managed Fastly CDN,
```
Comment (CollierCZ) [inline docs/src/domains/cdn/managed-fastly.md:18]: :pick: To increase directness and potentially avoid confusion with subscriptions.
```suggestion
To add a managed Fastly CDN to your project,
```

File: docs/src/domains/cdn/managed-fastly.md [hunk 29]
```diff
@@ -0,0 +1,48 @@
...
+[contact our sales team](https://platform.sh/contact/).
```
Comment (CollierCZ) [inline docs/src/domains/cdn/managed-fastly.md:19]: :clipboard: To avoid the first person.
```suggestion
[contact sales](https://platform.sh/contact/).
```

File: .platform/routes.yaml (hunk 26 above)
Comment (AnouckColson) [reply .platform/routes.yaml:143]: Looks like my brain had a glitch 🙈

**Review by CollierCZ — APPROVED** [review]
